ballerina/types.bal

```diff
@@ -0,0 +1,58 @@
+import ballerina/sql;
```

Comment:
Shall we add license headers

Reply:
Fixed in [3ac4df4](https://github.com/ballerina-platform/module-ballerina-persist/pull/155/commits/3ac4df4447e4c445dda487eb932865f44b7f670c)